### ISSUE TITLE
flatpak_package provider

### ISSUE THREAD

**Issue by elijah** [chef/chef#8760]
### Describe the Enhancement:

It would be useful to be able to automate and manage the installation of flatpak-format packages from flathub.

### Describe the Need:

We already have support in chef for ubuntu snap packages - these are very similar.  The flatpak utility is written by a Redhat employee - and in use on e.g. Fedora Silverblue.

### Current Alternative

Workaround is to use shell calls to install the package management utilities and then issue the proper shell commands to install the packages you want.

### Can We Help You Implement This?:

Not really a thing I can help with at this time.  But filing the request for enhancement so it's tracked.

**Comment by tas50** [comment]
Thanks for submitting this issue. After review by the team, we've decided that this isn't something that we're going to work on ourselves. We receive dozens of new issues a month, which is more than our team can possibly work on. This means we must prioritize work and be honest about what we're able to get done as a team.  That doesn't mean that we think this is a bad idea or isn't a necessary change. It's just not something we'll likely get to given competing priorities. I've applied our `Design Proposal: Accepted. PRs Welcome` label to this issue and closed it out. If you or someone else would like to work on this issue we're open to pull requests. If you'd like to chat further about what that design might look reach out to us in our [#chef-infra-dev](https://chefcommunity.slack.com/archives/C55FCUGBS) slack channel.
